### PR TITLE
StackSets - ListStackSetsRequest with ACTIVE status

### PR DESCRIPTION
**Pull request by xiwhuang**
*Issue #, if available:*

Preivously, we list all stacksets including `Deleted` Stacksets, that causes an unnecessary latency issue to users who have a lot of StackSets deleted before.

*Description of changes:*

Optimizing the List handler to call `ListStackSets` with `ACTIVE` status flag

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: aws-cloudformation-stackset/src/main/java/software/amazon/cloudformation/stackset/translator/RequestTranslator.java [hunk 1]
```diff
@@ -124,6 +125,7 @@ public static ListStackSetsRequest listStackSetsRequest(
         return ListStackSetsRequest.builder()
                 .maxResults(LIST_MAX_ITEMS)
                 .nextToken(nextToken)
+                .status(ACTIVE)
```
Comment (xalle-y) [inline aws-cloudformation-stackset/src/main/java/software/amazon/cloudformation/stackset/translator/RequestTranslator.java:128]: Are there any places where we may want to list deleted stacksets?

**Review by xalle-y — APPROVED** [review]

File: aws-cloudformation-stackset/src/main/java/software/amazon/cloudformation/stackset/translator/RequestTranslator.java (hunk 1 above)
Comment (xiwhuang) [reply aws-cloudformation-stackset/src/main/java/software/amazon/cloudformation/stackset/translator/RequestTranslator.java:128]: Not that I know, at least, currently the list stacksets are only used in list handler. which will just return a list of live StackSets

**Comment by ugudip** [comment]
Can we add more context on the description on what this PR is doing? Is this to list only active stacksets every time?

File: .github/workflows/java-ci-with-maven.yml [hunk 2]
```diff
@@ -30,7 +30,7 @@ jobs:
     - name: Install jinja2 with 2.11.3 to resolve incompatible issue
       run: pip install jinja2==2.11.3
     - name: Set up cloudformation-cli-java-plugin
-      run: pip install cloudformation-cli-java-plugin
+      run: pip install cloudformation-cli-java-plugin==2.0.3
```
Comment (ugudip) [inline .github/workflows/java-ci-with-maven.yml:33]: Spoke to @xiwhuang offline, the current java plugin is breaking the build checks, so downgrading it to 2.0.3 for now. However, we should revert this change once the latest version is working as expected because it's recommended to use the latest version.

**Comment by xiwhuang** [comment]
Sure. Not sure why the description was not added. Basically it is optimizing the impl in List handler.

**Review by ugudip — APPROVED** [review]
